### PR TITLE
New version: shirowanisan.COEIROINK version 2.3.4

### PR DESCRIPTION
**Pull request by Exorcism0666**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/138658)

### CODE REVIEW

File: manifests/s/shirowanisan/COEIROINK/2.3.4/shirowanisan.COEIROINK.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Created with komac v2.0.4
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: shirowanisan.COEIROINK
+PackageVersion: 2.3.4
+InstallerType: zip
+NestedInstallerType: exe
+NestedInstallerFiles:
+- RelativeFilePath: COEIROINK_WIN_CPU_v.2.3.3.exe
```
Comment (Exorcism0666) [inline manifests/s/shirowanisan/COEIROINK/2.3.4/shirowanisan.COEIROINK.installer.yaml:9]: @russellbanks The new exe is not detected by komac for each new update

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  Installation failed with exit code -1978335141 2024-02-10 08:39:30.645 [CORE] DeliveryOptimization error: 0x80190190, extended error: 0x00000000 2024-02-10 08:39:30.685 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\DODownloader.cpp(289)\WindowsPackageManager.dll!00007FF8B891A465: (caller: 00007FF8B891AD01) Exception(1) tid(1b74) 80190190 Bad request (400).

(Automated response - build 720.)

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping Exorcism0666.
>[!warning]
>With the new version of Komac, numerous Pull Requests may contain bugs or unwanted elements. If such issues arise, please tag russellbanks so that these bugs can be addressed in future updates.

>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

File: manifests/s/shirowanisan/COEIROINK/2.3.4/shirowanisan.COEIROINK.installer.yaml (hunk 1 above)
Comment (stephengillie) [inline manifests/s/shirowanisan/COEIROINK/2.3.4/shirowanisan.COEIROINK.installer.yaml:9]:
```suggestion
- RelativeFilePath: COEIROINK_WIN_CPU_v.2.3.4.exe
```

**Review by stephengillie — APPROVED** [review]

**Pull request merged**